form/dynamic_form_modification.rst

```diff
@@ -301,13 +298,13 @@ and fill in the listener logic::
                     $formOptions = array(
                         'class'         => User::class,
                         'choice_label'  => 'fullName',
-                        'query_builder' => function (EntityRepository $er) use ($user) {
+                        'query_builder' => function (EntityRepository $users) use ($user) {
```

Comment:
why not using userRepository ?

It feels strange for a `$users` variable to not contain a collection of user IMHO.

Reply:
This is a new practice that we're slowly introducing in Symfony (and commonly used in other popular projects). We use it for example in the Symfony Demo app: https://github.com/symfony/demo/blob/3c5d376ac8e3f7ae34d972ebb11de9542d0e73ff/src/Controller/BlogController.php#L51  Instead of `XxxxRepository $xxxRepository` we call it `XxxRepository $xxxs` because we consider that the code is easier to understand that way.

Reply:
I finally renamed the variable to `$userRepository` as you asked!